lib/graphiti/configuration.rb

```diff
@@ -43,7 +43,7 @@ def initialize
         end
 
         if (logger = ::Rails.logger)
-          self.debug = logger.level.zero?
+          self.debug = self.debug && logger.level.zero?
```

Comment:
Before: turn the debugger on if the Rails logger level is DEBUG. Ignore the users configuration.
After: turn the debugger on if the user hasn't explicitly disabled the debugger AND the Rails logger level is DEBUG.